docs/guidelines/language/operational-emails/general-information-and-administrative-emails.md

```
@@ -0,0 +1,189 @@
+---
+sidebar_position: 7
+sidebar_label: General information and administrative emails
+title: General information and administrative emails
+doc-type: 'banner'
+description: 'In everyday operations, clear communication is key to maintaining transparency and trust. General notifications - covering administrative details, confirmations, and other essential updates outside specific operational categories - help keep stakeholders informed and processes organized. Well-structured messaging ensures clarity, reduces confusion, and supports smooth business interactions.'
+---
+
+#
+
+Note: Use these rules alongside the [General rules](#) and we have used [App name] as the standard placeholder for you to adapt individually to your product. All placeholders are shown with square brackets [placeholder].
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)